src/utils/metadata.ts

```diff
@@ -0,0 +1,126 @@
+
```

Comment:
Everything in this file is too project specific. We will accept it for now but will refactor it immediately for the next release. You cannot deal with specific keys in metadata. You have to expect the keys to be **arbitrary**.